Floofbot/Modules/Help.cs

```diff
@@ -73,5 +75,74 @@ public async Task HelpCommand()
                 Trash = true
             });
         }
+
+        [Summary("Show help for a specific module")]
+        [Command("help")]
+        public async Task HelpCommand([Summary("module")]string requestedModule)
+        {
+            try
+            {
+                List<string> moduleNames = new List<string>();
+                List<CommandInfo> commands = _commands.Commands.ToList();
+                List<EmbedFieldBuilder> fields = new List<EmbedFieldBuilder>();
+                List<PaginatedMessage.Page> pages = new List<PaginatedMessage.Page>();
+
+                // put available module names into a list instead of their object
+                foreach (ModuleInfo module in _commands.Modules.ToList())
+                    moduleNames.Add(module.Name.ToLower());
+
+                if (!moduleNames.Contains(requestedModule.ToLower())) // no such cmd
```

Comment:
We can probably save a few lines here using .Any() from LINQ if you wanna be fancy. 
https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.any?view=netcore-3.1#System_Linq_Enumerable_Any__1_System_Collections_Generic_IEnumerable___0__System_Func___0_System_Boolean__
What we have here is fine, though

Reply:
I believe this can be refactored in a future PR if required